contracts/Penalizer.sol

```diff
@@ -1,19 +1,25 @@
+/* solhint-disable avoid-low-level-calls */
+/* solhint-disable no-inline-assembly */
 // SPDX-License-Identifier:MIT
 pragma solidity ^0.6.12;
 pragma experimental ABIEncoderV2;
 
+import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
OpenZepellin usually comes with lots of code that is not needed, making contracts heavier.
I would suggest implementing ownable as we do in other contracts, instead of using OZ. But it is not mandatory, just a suggestion